handlers/main.yml

```diff
@@ -1 +1,6 @@
 ---
+
+- name: restart php-fpm
```

Comment:
Adjust the handler name as defined in our standard (https://docs.adfinis-sygroup.ch/public/ansible-guide/styling_guide.html).